docs/test/actions.md

```diff
@@ -1262,6 +1262,7 @@ Attribute|Type|Use|Description
 ---|---|---|---
 `command`|string |optional| CLI command to be executed in Magento environment.
 `arguments`|string |optional| Unescaped arguments to be passed in with the CLI command.
+`timeout`|string|optional| Number of seconds CLI command can run without outputting anything.
```

Comment:
Can we add `magentoCLISecret` to the documentation too?

Reply:
`magentoCLISecret` is not an exposed action, it's internally mapped to magentoCLI based on references to _CREDS; may not need documentation imo.

Reply:
Makes sense to me. No action needed.